rct229/rulesets/ashrae9012019/section5/section5rule1.py

```diff
@@ -112,7 +112,7 @@ def create_data(self, context, data):
 
             # filter out baseline rmds that aren't None then count the length
```

Comment:
The comment does not seem the be correct. This includes baseline and proposed.

Reply:
Addressed.